api/pkg/cli/hub/hub.go

```diff
@@ -33,6 +33,12 @@ const (
 	tektonHubURL   = "https://api.hub.tekton.dev"
 	artifactHubURL = "https://artifacthub.io"
 	hubConfigPath  = ".tekton/hub-config"
+
+	tektonHubCatEndpoint         = "/v1/catalogs"
+	artifactHubCatSearchEndpoint = "/api/v1/repositories/search"
+	artifactHubCatInfoEndpoint   = "/api/v1/packages/tekton"
+	artifactHubTaskType          = 7
+	artifactHubPipelineType      = 11
```

Comment:
I don't remember why we need this ?

Reply:
Artifact Hub hosts ~10 kinds of resources. Tekton Task and Tekton Pipelines are two of the kinds:

https://artifacthub.io/docs/api/#/Packages/searchPackages

Reply:
And this number will remain constant for Task and Pipeline right ?

Reply:
Yes, you are right.